Ldap/LdapManager.php

```diff
@@ -42,7 +37,7 @@ public function findUserByUsername($username)
     public function findUserBy(array $criteria)
     {
         $filter  = $this->buildFilter($criteria);
-        $entries = $this->driver->search($this->params['baseDn'], $filter, $this->ldapAttributes);
```

Comment:
Thats a good thing but I am not sure this will work in combination with memberOf informations. As I know OpenLdap for example requires the definition for memberof as requested attribute. Otherwise no memberof info will be returned. I test this in the next days to be sure.